.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "disabled"
```

Comment:
Let's keep the repo clean of editor-specific config files, and add this to .gitignore.

Reply:
sure, makes sense to me!